cerbero/packages/wix_packager.py

```diff
@@ -121,8 +128,8 @@ def _is_app(self):
             isinstance(self.package, AppExtensionPackage)
 
     def _package_name(self, version):
-        return "%s-%s-%s" % (self.package.name, version,
-                             self.config.target_arch)
+        return "%s-%s-%s" % (self.package.name, self.config.target_arch,
```

Comment:
OK for that change, but it's probably going to break other things like the release upload scripts

Reply:
The reasoning was to have the same name schema as other packages, I'll modify the upload script then too, wasn't there when I did this patch :)

Reply:
Looking at the code, the change was done on the merge module packager, there's no need to change the upload script